drupal8-apache/7.0/usr/local/share/drupal8/install_finalise.sh

```diff
@@ -20,9 +20,14 @@ chmod u+w "$SETTINGS_DIR" || true
 mkdir -p "$SETTINGS_DIR/files/"
 
 # Install a database if there isn't one yet
-CURRENT_TABLES="$(as_build "drush sql-query 'SHOW TABLES;'" /app/docroot)"
-if [ "$CURRENT_TABLES" == '' ] && [ -n "$DRUPAL_INSTALL_PROFILE" ]; then
-  as_build "echo 'y' | drush site-install '$DRUPAL_INSTALL_PROFILE'" "/app/docroot"
+set +e
+as_code_owner "drush sql-query 'SHOW TABLES;' | grep -v cache | grep -q ''" /app/docroot
+HAS_CURRENT_TABLES=$?
```

Comment:
is there any way to do this that wont run site-install if there are connectivity issues?